### PR TITLE
Preparation for release v2023.2 (Zonemaster/Zonemaster)

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

Preparation for release.

It is assumed that #1240 is approved and included in the release.

## Changes

Updates files `Changes` and `zonemaster-gui.md` (installation).

## How to test this PR

Review the changes.

### CODE REVIEW

**Review by marc-vanderwal — APPROVED** [review]

File: Changes [hunk 1]
```diff
@@ -1,5 +1,50 @@
...
+ - Adds documentation of global cache (#1227, #1240)
+
+ [Fixes]
+ - Updates implementation of test cases in Zonemaster-Engine.
+ - Updates specification of test cases DNSSEC05, DNSSEC03
+   and Nameserver15 (#1183, #1189, #1199)
+ - Adds updated Arguments document (#1190)
+ - Adds specification of test zones for test cases Consistency05,
+   Consistency06, Nameserver15 and DNSSEC03 (#1213, #1217, #1218)
+ - Updates installation instructions (#1223, #1225, #1236, #1239
```
Comment (marc-vanderwal) [inline Changes:30]:
```suggestion
 - Updates installation instructions (#1223, #1225, #1236, #1239)
```

**Review by marc-vanderwal — COMMENTED** [review]
Also, the version of Zonemaster-GUI needs to be bumped to 4.2.0.

**Comment by matsduf** [comment]
@marc-vanderwal, @mattias-p and @tgreenx, please review.

**Comment by matsduf** [comment]
> Also, the version of Zonemaster-GUI needs to be bumped to 4.2.0.

Done.

**Review by marc-vanderwal — DISMISSED** [review]

File: Changes [hunk 2]
```diff
@@ -1,5 +1,51 @@
 Release history of Zonemaster
 
+
+v2023.2 2024-03-18 (public release version)
```
[the remark this answers is not in the document]
Comment (matsduf) [reply Changes:4]: I do not remember, but fine for me.

**Pull request merged**